ChangeLog

```diff
@@ -18,18 +56,75 @@
+	  classes (CFE-3140)
+	- Version specific distro classes are now collected by default in
+	  Enterprise (ENT-4752)
+	- redhat_8 and redhat_8_0 are now defined on RHEL 8 (CFE-3140)
+	- Added derived-from-file tag to hard classes based on /etc/redhat-release
+	  (CFE-3140)
+	- Added sys.bootstrap_id policy variable Contains the ID from
+	  `/var/cfengine/bootstrap_id.dat`, if present (CFE-2977)
+	- sys.interfaces now contains interfaces even when they only have 
+	  IPv6 addresses (ENT-4858)
+	- IPv6-only interfaces added to sys.hardware_(addresses,mac) (CFE-3164)
+	- IPv6 addresses are now added to policy variable sys.ip_addresses
+	  (CFE-682)
+	- IPv6 addresses now respect ignored_interfaces.rx (CFE-3156)
+	- hostname now allowed in bindtoaddress (CFE-3190)
+	- Fixed issue when removing comments from files in various policy functions
+	  This also fixes many erroneous occurences of the error message
+	  mentioning:
+	  ```
+	     [...] because it legally matches nothing
+	  ```
+	  (A warning can still appear if a comment regex actually matches nothing).
+	  Also made this comment removing logic faster.
+	  Affected functions include:
+	  * readstringlist()
+	  * readintlist()
+	  * readreallist()
+	  * peers()
+	  * peerleader()
+	  * peerleaders()
+	  * data_readstringarray()
+	  * data_readstringarrayidx()
+	  * data_expand()
+	  * readstringarray()
+	  * readstringarrayidx()
+	  * readintarray()
+	  * readrealarray()
+	  * parsestringarray()
+	  * parsestringarrayidx()
+	  * parseintarray()
+	  * parserealarray()
+	  (CFE-3188, ENT-5019)
+	- Fixed memory leak in JSON / env file parsing (CFE-3210)
+	- Fixed memory leak in handling of nfs / fstab (CFE-3210)
+	- Fixed memory leak in string_replace() and regex_replace() (CFE-3210)
+	- Fixed memory leak when using `with` constraint (CFE-3210)
+	- Fixed minor memory leak in policy evaluation (CFE-3210)
+	- Fixed small memory leak in SQL database promises (CFE-3210)
+	- Received SIGBUS now triggers a repair of local DBs (CFE-3127)
+	- Corrupted LMDB files are now automatically repaired (CFE-3127)
+	- Keys in the lock database, cf_lock.lmdb, are now human-readable
+	  (CFE-2596)
+	- Local databases now use synchronous access on AIX and Solaris (ENT-4002)
+	- Report corrupted local database with a critical log message (CFE-2469)
+	- Local DB errors are now logged with the particular DB file path (CFE-2469)
+	- cf-check: dump command now prints structs as JSON objects (CFE-2989)
+	- cf-check: repair now preserves readable data in corrupted LMDB files
+	  (CFE-3127)
+	- cf-check: --dump option was added to the backup command
 	- cf-check: Added --no-fork to diagnose command (CFE-3145)
 	- cf-check: Added -M manpage option and other common options (CFE-3082)
 	- cf-check: No DB files in state dir now causes errors
 	- cf-check: dump command now dumps DB contents to JSON5 (CFE-3126)
 	- cf-check: help command can now take a topic as argument
```

Comment:
So CFE-2989 above says "as JSON" and CFE-3126 says "JSON5" so I assume now we have JSON5 output. Is that documented somewhere? It seems that the cf-check doc/man page doesn't mention it. Seems like it would be nice to highlight that up-front. 

https://docs.cfengine.com/docs/master/reference-components-cf-check.html

Reply:
Deleted redundant entry.